tests/test_selectors.py

```diff
@@ -106,3 +106,30 @@ def test_connection_error(self):
             get_bug.side_effect = requests.exceptions.ConnectionError
             with self.assertWarns(RuntimeWarning):
                 selectors.bug_is_fixed(None, ver)
+
+
+class SkipIfConditionTestCase(unittest.TestCase):
+    """Test :meth: `pulp_smash.selectors.skip_if`."""
+
+    class _DummyClass:  # pylint:disable=missing-docstring,too-few-public-methods
+
+        def __init__(self):
+            self.element = 'test'
+
+        @selectors.skip_if(bool, 'element', True, unittest.SkipTest)
+        def dummy_should_skip(self):  # pylint:disable=missing-docstring,no-self-use
+            pass
+
+        @selectors.skip_if(bool, 'element', False, unittest.SkipTest)
+        def dummy_should_run(self, bool_val):  # pylint:disable=missing-docstring,no-self-use
+            return bool_val
+
+    def test_skip_true(self):
+        """Assert the skip_if decorator skips test if conditions is satisfied."""
+        self.assertRaises(unittest.SkipTest, self._DummyClass().dummy_should_skip)
+
+    def test_skip_false(self):
+        """Assert the skip_if decorator doesn't skips test if conditions is not satisfied."""
+        for test_val in (True, False):
```

Comment:
This for loop doesn't help to prove that `@skip_if` works correctly, and it does make the test method more complicated.